src/contexts/InjectiveWalletContext.tsx

```diff
@@ -1,127 +1,53 @@
+import { useMemo } from "react";
 import {
-  createContext,
-  ReactChildren,
-  useCallback,
-  useContext,
-  useMemo,
-  useState,
-} from "react";
-import { Wallet, WalletStrategy } from "@injectivelabs/wallet-ts";
-import keplrIcon from "../icons/keplr.svg";
-import { getInjectiveNetworkChainId } from "../utils/consts";
-
-interface IInjectiveProviderContext {
-  connect(wallet: Wallet): void;
-  disconnect(): void;
-  wallet: WalletStrategy | null;
-  address: string | null;
-}
-
-const InjectiveProviderContext = createContext<IInjectiveProviderContext>({
-  connect: (wallet: Wallet) => {},
-  disconnect: () => {},
-  wallet: null,
-  address: null,
-});
+  getInjectiveNetworkChainId,
+  getInjectiveNetworkName,
+} from "../utils/consts";
+import {
+  InjectiveWallet,
+  KeplrWallet,
+} from "@xlabs-libs/wallet-aggregator-injective";
+import { useWallet } from "@xlabs-libs/wallet-aggregator-react";
+import { CHAIN_ID_INJECTIVE } from "@xlabs-libs/wallet-aggregator-core";
+import { getNetworkInfo } from "@injectivelabs/networks";
+
+export const getInjectiveWallets = () => {
+  if (!["mainnet", "testnet"].includes(process.env.REACT_APP_CLUSTER || ""))
```

Comment:
This validation can be extract to a `variable`

Reply:
Do you mean extract the `["mainnet", "testnet"]` to a constant in the outer scope?

Reply:
Ah! I see now. I think it's clearer that way, thanks for pointing it out.